src/utils/MySQLDataHandler.py

```diff
@@ -393,13 +354,25 @@ def UpdateStorage(self, vcName, url, storageType, metadata, defaultMountPath):
             logger.error('Exception: %s', str(e))
             return False
 
-
     @record
-    def AddVC(self, vcName, quota, metadata):
+    def init_vc_sqls(self, config):
+        ratio_dict = {config['defalt_virtual_cluster_name']: 1.0}
+        if "vc_resource_ratio" in config:
+            ratio_dict = config["vc_resource_ratio"]
+        quota, metadata, res_quota, res_meta = vc_value_str(config, ratio_dict)
+        if quota == "":
+            return
+        sqls = []
+        for i, (vc, vc_res_quota) in enumerate(res_quota.items()):
```

Comment:
`i` is not used. The for loop can be simplified to `for vc, vc_res_quota in res_quota.items():`